go.mod

```diff
@@ -1,36 +1,36 @@
 module github.com/openshift/rbac-permissions-operator
 
-go 1.19
+go 1.20
 
 require (
 	github.com/golang/mock v1.6.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/ginkgo/v2 v2.9.7
-	github.com/onsi/gomega v1.27.7
+	github.com/onsi/ginkgo/v2 v2.13.0
+	github.com/onsi/gomega v1.29.0
 	github.com/openshift/operator-custom-metrics v0.4.3-0.20220322205053-7b528cc0d6eb
 	github.com/openshift/osde2e-common v0.0.0-20240111205449-2d6094e0e389
 	github.com/operator-framework/operator-lib v0.10.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.62.0
 	github.com/prometheus/client_golang v1.15.1
 	github.com/sykesm/zap-logfmt v0.0.4
 	go.uber.org/zap v1.24.0
-	k8s.io/api v0.28.2
-	k8s.io/apimachinery v0.28.2
-	k8s.io/client-go v0.28.2
+	k8s.io/api v0.29.2
+	k8s.io/apimachinery v0.29.2
+	k8s.io/client-go v0.29.1
```

Comment:
Can you align it to 0.28.2 as per https://github.com/openshift/api/blob/release-4.15/go.mod#L8 ? Let's tie the version to 4.15 OCP release branch

Reply:
Thank you Ravi for the review. I have aligned dependencies accordingly